js/windowMgr.js

```diff
@@ -70,6 +70,7 @@ const DEFAULT_HEIGHT = 600;
 
 // Certificate transparency whitelist
 let ctWhitelist = [];
+let ignoreAllCertErrors = false;
```

Comment:
I think we should reset this to `true` if the browser window is refreshed because that's the behaviour on Chrome as well.